assets/stylesheets/units/_headings.scss

```diff
@@ -2,7 +2,8 @@ h1 {
   @include bold-font(64);
 }
 
-h2 {
+h2,
+.heading__level-2 {
```

Comment:
As I mentioned earlier, if we're having font size modifiers I think they should be decoupled from being specific to a heading level. We might decide to change the size of this heading level but where the modifier is used it might still need to be the size set based on what this heading level originally was?